.pytool/Plugin/SpellCheck/Readme.md

```diff
@@ -45,6 +46,17 @@ This supports .gitignore file and folder matching strings including wildcards
 This list allows words to be added to the dictionary for the spell checker when
 this package is tested.  These follow the rules of the cspell config words field.
 
+### ExtraDictionaries
+
+This list allows dictionary files to be added to the dictionary for the spell
+checker when this package is tested. Dictionary files contain one word per line.
+Dictionary paths interpreted from workspace root. All words in custom dictionary

```

Comment:
The other file paths are all package relative so that these are more portable.  Is there a reason you made this workspace relative?  I would prefer to be consistent and allow package portability.

Reply:
I made the dictionary files workspace relative because there are many common words used in lots of places in the code. Similar reasoning, I imagine, as to why there is a list of ~265 words to add to the dictionary in the plugin's default config: they aren't spelling mistakes and having to re-explain to the tool in each new package that `guids` should still be accepted as a word is a lot more work to maintain. It also makes integration with vs code's plugin easier since you don't have to add and maintain a list of all the dictionary files.

Maybe using `GetAbsolutePathOnThisSystemFromEdk2RelativePath` would be more flexible? Though I can imagine that silently using a dictionary file other than what the user intended.

Reply:
Is there the sample usage for the updated tool?